internal/target/service_list_response.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package target
+
+import (
+	"github.com/hashicorp/boundary/internal/refreshtoken"
+)
+
+// will likely eventually become a generic response and moved from target
+type ListResponse struct {
+	Items               []Target
+	DeletedIds          []string
+	EstimatedTotalItems int
+	CompleteListing     bool
+	RefreshToken        *refreshtoken.RefreshToken
+}
```

Comment:
Should this be in its own file?

Reply:
If it is in its own file, I don't think the file should start with `service_`. I think we should reserve that for files that contain service implementations, not just types.